p2p/test/ipfs.go

```diff
@@ -41,6 +41,7 @@ func MakeRepoFromIPFSNode(node *core.IpfsNode, username string) (qrirepo.Repo, e
 	}
 
 	memFS := qfs.NewMemFS()
+	// TODO(arqu) - verify if we need cafs/local here
```

Comment:
Was unsure on how/wether to handle this. I know we want to move away from cafs anyways.

Reply:
Branch I'm working on to upgrade qri to the latest qfs makes these changes. I would leave this unless tests are failing.